lnd.go

```diff
@@ -324,6 +327,14 @@ func Main(cfg *Config, lisCfg ListenerCfg, interceptor signal.Interceptor) error
 		return err
 	}
 
+	// If we're started in auto unlock mode, then a wallet _must_ already
+	// exist because we never want to enable the RPC unlocker in that case.
+	if cfg.WalletUnlockPasswordFile != "" && !walletExists {
```

Comment:
So with this users will need to do the normal RPC init once, write that output to a file (in the expected format), _then_ restart again and set the password unlock file?

Reply:
Yes, for now. With https://github.com/lightningnetwork/lnd/pull/5150 the wallet DB could be created outside of `lnd` prior to it being started (using `lndinit`). I described the process in a new doc. We can extend that doc if we decide to implement `lndinit`.